README.md

```diff
@@ -63,14 +61,14 @@ For CLI (command-line interface) rendering install
 % openjscad logo.jscad -of amf                       # -- convert logo.jscad into logo.amf
 ```
 
-### Node module use
+### Use with Node Modules
 
 >Note: Please try to use a recent , LTS version of Node.js ie Node.js 6.x.x,
 node V4.x.X is also supported but not earlier versions (they are not maintained anymore)
 (see [here for more details](https://github.com/nodejs/LTS))
 
 ```
-npm install --save Spiritdude/OpenJSCAD.org
+npm install --save @jscad/OpenJSCAD.org
 ```
 
 and then simply import and use openjscad:
```

Comment:
I need to verify this , and check if all is written down here (it should match what is in the tests)
VERY IMPORTANT: I think we should include a paragraph and examples here about the fact that you can use small parts of code 'to your taste' thanks to our modular approach with the small browser based / node.js example I had